tests/unit/tools/cargo_audit/test_cargo_audit_plugin.py

```
@@ -283,3 +302,29 @@ def test_check_timeout(
+
+    garbage = "this is not json {["
+
+    with patch(
+        "lintro.plugins.execution_preparation.verify_tool_version",
+        return_value=None,
+    ):
+        with patch.object(
+            cargo_audit_plugin,
+            "_run_subprocess_result",
+            return_value=_proc(success=True, stdout=garbage),
+        ):
```

Ruff (0.15.20)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)